Dockerfile.machine-config-daemon.upstream

```diff
@@ -0,0 +1,10 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/machine-config-operator
+COPY . .
+RUN WHAT=machine-config-daemon ./hack/build-go.sh
+
+# Temporarily until the UBI base allows us to install util-linux without entitlements
```

Comment:
This has nothing to do with UBI.  This is because we only install one repo mirror that corresponds to our working set in CI.  You can get this working locally by downloading:

https://github.com/openshift/release/blob/master/cluster/ci/config/prow/openshift/rpm-mirrors/ocp-4.0-default.repo
https://github.com/openshift/shared-secrets/blob/master/mirror/ops-mirror.pem

and putting them in the same directory, then using -v to imagebuilder or podman.

Reply:
I sent an email with instructions, you shouldn't need to add this.

Reply:
Fixing this, thanks Clayton for pointing this out